.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+## Summary
+
+Please describe your changes and what you plan to introduce as part of this pull request submissions. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.
+
+## Pull Request Checklist
+🚨Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.
+
+- [ ] Please ensure that you are making a pull request against the **Development** branch
+- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
+- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
+- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
+- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
+- [ ] Please include any relevant screenshots and/or output as part of your testing
+- [ ] Please include any documentation updates that is applicable for your changes
+
+## Change Type
+
+What types of changes does your code introduce to the VMware Event Broker Appliance?
+
+_Put an `x` in all boxes that apply_
+
+Please check the type of change your PR introduces:
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] Feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
```

Comment:
could code style update be bundled w/ refactoring or be removed as an option? I'm hoping we avoid PRs with only formatting changes and no additional value to the product. Maybe you have some example in mind that i'm overlooking.

Reply:
There's been a few cases where renaming has help clarified things, obviously that was part of a larger enhancement but I could see that happen independent of a feature. I also remember there were cases where we had some formatting issues that wasn't ideal. It wasn't so much "refactoring" in the traditional sense. We could certainly combine them but I also got the idea from other PR template examples

Reply:
thinking through this and from your explanation, i think this is good as is. If a name change is warranted along with other changes, it gives the Reviewer an headsup on all the changes that were submitted as a part of the PR and helps speed up the process.